src/ui/include/nqqsettings.h

```diff
@@ -63,6 +63,7 @@ class NqqSettings {
 public:
 
     BEGIN_GENERAL_CATEGORY(General)
+        NQQ_SETTING(FirstRun,                       bool,       true)
         NQQ_SETTING(Localization,                   QString,    "en")
```

Comment:
Instead of a "FirstRun" setting, can't you change the default value for "Localization" to an empty string and check if that is empty?